lib/s3_uploader.rb

```diff
@@ -0,0 +1,21 @@
+require 'aws-sdk'
+
+class S3Uploader
+  def self.upload(file, path)
+    s3_file = s3_bucket.objects[path].write(file: file)
+    s3_file.acl = :public_read
+
+    s3_file
+  end
+
+  def self.s3_bucket
+    bucket = s3.buckets[ENV['AWS_S3_BUCKET']]
+    bucket.acl = :public_read
```

Comment:
Oh this is definitely bad as you are setting the entire bucket to default to public so all files inside it would be public